### PR TITLE
More team name changes

### PR DESCRIPTION
**Pull request by JimMH**
Properly change the Moderator Team to Community Team now on the Get Involved page. 
Previous PR was on the old website repo instead of the Symfony site, that's fixed now.

Also change many other references of the Extensions Team to Extensions Customisations Team and change modifications to extensions on our homepage and several landing pages.

### CODE REVIEW

**Comment by DavidIQ** [comment]
Looks like tests need updating. Otherwise +1 from me.

**Comment by JimMH** [comment]
After some attempts, tests are fixed now.

**Comment by vinny** [comment]
@JimMH Could you include [this](https://github.com/phpbb/phpbb-website/blob/master/src/AppBundle/Resources/views/Styles/team-overview.html.twig#L29) change? I forgot in my last PR.

**Comment by JimMH** [comment]
@vinny Sure, that's included now!

**Comment by vinny** [comment]
Thanks :)

File: src/AppBundle/Resources/views/About/getInvolved.html.twig [hunk 1]
```diff
@@ -75,26 +75,26 @@
 		<li>Documentation: <a href="{{ SUPPORT_DOCS_CONTACT }}">{{ SUPPORT_DOCS_NAME }}</a></li>
 	</ul>
 
-	<h3 class="teamtitle" style="color: rgb(102, 0, 153);">Extensions Team</h3>
-	<p class="get-involved">The primary task of the Extensions Team is to validate extensions submitted through the Customisation Database and manage the extension related tools provided for extension authors and users. The Extensions Team is also responsible for the extensions forums and for the upkeep of extension documentation.</p>
+	<h3 class="teamtitle" style="color: rgb(170, 0, 0);">Extension Customisations Team</h3>
+	<p class="get-involved">The primary task of the Extension Customisations Team is to validate extensions submitted through the Customisation Database and manage the extension related tools provided for extension authors and users. The Extension Customisation Team is also responsible for the extensions forums and for the upkeep of extension documentation.</p>
```
Comment (DavidIQ) [inline src/AppBundle/Resources/views/About/getInvolved.html.twig]: `Extension Customisations Team` (for the second instance)

File: src/AppBundle/Resources/views/About/getInvolved.html.twig [hunk 2]
```diff
@@ -75,26 +75,26 @@
...
+	<p class="get-involved">The Extension Customisation Team watches for talented individuals who are active in the extensions forums and who have submitted an extension which has successfully passed validation for all invitations. The Jr. Validation Team and Extension Development Team are the main entry points for new members on the Extension Customisations Team. Those on the <a href="{{ extensions_junior_validators }}">Jr. Validation Team</a> are tasked with assisting validators on the Extension Customisations Team to help smooth and speed up the validation process. Those on the <a href="{{ extensions_offical_exts_team }}">Extension Development Team</a> work under the mentorship of the Extension Customisations Team to create and maintain phpBB's official extensions. The Jr. Validation and Extension Development Teams accept applications to join, if you are interested in joining you may apply by visiting their respective pages.</p>
```
Comment (DavidIQ) [inline src/AppBundle/Resources/views/About/getInvolved.html.twig]: `The Extension Customisations Team`

File: src/AppBundle/Resources/views/About/getInvolved.html.twig [hunk 3]
```diff
@@ -75,26 +75,26 @@
...
+	<h3 class="teamtitle" style="color: rgb(0, 0, 204);">Community Team</h3>
+	<p class="get-involved">The Community Team members at phpBB are part Ambassador, part Peace Officer. They keep the forums running smoothly by directing questions to appropriate locations, approving posts, and answering basic support questions. Community Team members perform all basic moderation tasks within all forums, but leave the other teams to enforce rules specific to their respective forums. They also ensure members are adhering to site wide rules, and they will be the ones to contact you should you need a reminder about the rules. Moderator team members welcome all Private Messages regarding rules or how to use this site itself, but regret that they cannot answer support questions privately. Our Community Team covers the forums at www.phpbb.com/community as well as the Area51 forums and our IRC channel.</p>
```
Comment (DavidIQ) [inline src/AppBundle/Resources/views/About/getInvolved.html.twig]: ~~Moderator team members~~ -> Community Team members

**Pull request merged**